### ISSUE TITLE
Error: invalid literal for int()

### ISSUE THREAD

**Issue by Nyx0uf** [ratoaq2/pgsrip#3]
Hi,

I have a mkv file with 2 PGS tracks (eng & fre). When trying to convert the subtitles I have the following error (for both tracks)

**Error while tyring to rip <mkv / or sup file>: <ValueError> [invalid literal for int() with base 10: '95.936287']**

**Comment by Victorbry99** [comment]
I get the same error ```invalid literal for int() with base 10```.
I didn't have that error before, maybe it's because I updated to python 3.10 (?). what python version do you have?

**Comment by Nyx0uf** [comment]
Yes I'm running Python 3.10.2

**Comment by Victorbry99** [comment]
me python 3.10.3.
here is the full debug log for the dev, in case he needs it:
```1 PGS subtitle collected from 1 file
WARNING:pgsrip.core:Error while trying to rip a.sup: <ValueError> [invalid literal for int() with base 10: '82.661789']

Traceback (most recent call last):
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/pgsrip/core.py", line 65, in rip_pgs
    srt = PgsToSrtRipper(pgs, options).rip(lambda t: rules.apply(t, '')[0])
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/pgsrip/ripper.py", line 191, in rip
    items = self.process(subs, items, post_process, confidence, max_width)
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/pgsrip/ripper.py", line 138, in process
    data = TsvData(tess.image_to_data(full_image.data, **config))
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/pgsrip/tsv.py", line 47, in __init__
    items = [TsvDataItem(**{k: values[i] for (i, k) in enumerate(keys)}) for values in (
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/pgsrip/tsv.py", line 47, in <listcomp>
    items = [TsvDataItem(**{k: values[i] for (i, k) in enumerate(keys)}) for values in (
  File "/data/data/com.termux/files/usr/lib/python3.10/site-packages/pgsrip/tsv.py", line 19, in __init__
    self.conf = int(conf)
ValueError: invalid literal for int() with base 10: '82.661789'```

**Comment by Victorbry99** [comment]
@Nyx0uf if yout git clone the project ,than edit line 19 of tsv.py ```self.conf = int(conf)``` with ```self.conf = int(float(conf))``` , then you install it with ```pip install .``` inside the first pgsrip folder, it will rip the pgs subs with no ValueError appearing, though Idk if it is a proper fix. I'm not a dev.

**Comment by macro1** [comment]
Seems it's [fixed in `pytesseract` 0.3.10](https://github.com/madmaze/pytesseract/commit/84e6c2b8c403dbb309ea3c3ad23b00c5da06827f) but they haven't published to PyPI yet per https://github.com/madmaze/pytesseract/issues/417

Either converting to float instead of forcing to int or @Victorbry99 seem like good options. `pytesseract` is resolving by converting to int anyway.
